### ISSUE TITLE
Lewis and Short: carets in many entries; misreading of double diacritic mark

### ISSUE THREAD

**Issue by ids1024** [PerseusDL/lexica#41]
For example academia appears as `&abreve;c&abreve;d&emacr;m&imacr;^a`.

![academia](https://user-images.githubusercontent.com/2263150/29956935-51e43cc6-8e9f-11e7-936b-dfb2ec96cf90.jpg)

I noticed this running a script to test if the key matches the first word of the entry (after stripping accents, etc.). It should be possible to write a script that fixes this; I don't think there should be (m)any false positives.

**Comment by lcerrato** [comment]
Hi. 
The lexica were encoded prior to there being Unicode equivalents for the various accents and stress marks so there are various idiosyncrachies throughout. The caret was intentional as a stopgap. Neither of the large lexica have been converted to Unicode yet.

**Comment by ids1024** [comment]
Fair enough. Assuming the caret always represents the same thing, it shouldn't be too hard to automatically script this.

Would a PR applying such a change be accepted, or are there issues to address first, etc?

**Comment by lcerrato** [comment]
All of the entities need conversion to Unicode and there are probably similar encoding gaps, etc. Then the markup itself needs conversion.  The primary sources are the priority at this writing. 
As noted in the wiki, none of the changes made here will be visible in Perseus in the short term, but we have been accepting pull requests on these versions.
